lesson-5/script.js

```diff
@@ -0,0 +1,104 @@
+// //Вивести в console числа, від 10 до 25 з кроком 0.5 (10 й 25 включно)
+for (i = 10; i <= 25; i += 0.5) {
```

Comment:
```suggestion
for (let i = 10; i <= 25; i += 0.5) {
```

Reply:
fixed. but Ros didn't used let or used it rarely